Sources/API/Operator.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Operator.swift
+//  SwiftPrettyPrint
+//
+//  Created by Yusuke Hosonuma on 2020/03/06.
+//
+
+// ----------------------------------------------------------------------------
+//
+// Provide a simplicity description for print.
+//
+// No need to enclose in parenthese unlike `print()` or `debugPrint()`.
+//
+// For example:
+//
+// ```
+// let string = "Hello, world."
+// string.hasPrefix("Hello")
+//
+// // when use `print()` - needs enclose in parenthese
+// Debug.print(string.hasPrefix("Hello"))
+//
+// // when use `p` - not needs enclose in parenthese
+// Debug.p >>> string.hasPrefix("Hello")
+// ```
+//
+// Also the operator `>>>` has lower precedence than all standard operators in Swift,
+// Therefore could be a apply to expression too.
+//
+// ----------------------------------------------------------------------------
+
+extension Debug {
+    public struct P {}
+    public struct PP {}
+    public struct PD {}
+    public struct PPD {}
+
+    public static var p: P { P() }
+    public static var pp: PP { PP() }
+    public static var pd: PD { PD() }
+    public static var ppd: PPD { PPD() }
+}
+
+precedencegroup PrintPrecedence {
+    lowerThan: FunctionArrowPrecedence
```

Comment:
Has lower precedence than all standard operators in Swift.
https://github.com/apple/swift/blob/e6f8843c701a0ea72ddfe0b5538b815767a4ee5d/stdlib/public/core/Policy.swift#L536-L582